src/chainweb.js

```diff
@@ -434,7 +434,7 @@ const chainUpdates = (depth, chainIds, callback, network, host) => {
     let bs = {};
     chainIds.forEach(x => bs[x] = new HeaderBuffer(depth, callback));
     return headerUpdates(
-        hdr => bs[hdr.header.chainId]?.add(hdr),
+        hdr => (bs[hdr.header.chainId].add === null || bs[hdr.header.chainId].add === undefined) ? undefined : bs[hdr.header.chainId].add(hdr),
```

Comment:
I think, this implements exactly the semantics of `?.`. I think, `?.` is a primitive javascript operator. How can that cause issues?

Reply:
because webpack is comprised mostly of strange matter found only in the outer darkness